src/universal/components/Dashboard/DashLayout.js

```diff
@@ -0,0 +1,64 @@
+import React, {PropTypes} from 'react';
+import look, {StyleSheet} from 'react-look';
+
+import Helmet from 'react-helmet';
+import {head} from 'universal/utils/clientOptions';
+
+import NotificationBar from 'universal/components/NotificationBar/NotificationBar';
+
+let styles = {};
+
+const DashLayout = (props) => {
```

Comment:
I propose we change the eslint to `"as-needed"`: http://eslint.org/docs/rules/arrow-parens
That way, when arity is 1, you don't have parens around the args.
I'm open to the opposite too, as long as we're consistent.